docs/tutorial-solver.md

```diff
@@ -4,6 +4,8 @@ title: Run a Solver
 sidebar_label: Overview
 ---
 
+<!-- @format -->
```

Comment:
Just curious, what's this line for?

Reply:
Unsure, I didn't include that intentionally. Must have been my IDE.

Reply:
Turns out my IDE includes this line on every save for some reason... would it be ok to leave it here for now until I figure out whats up with my auto-formater?